packages/elements/src/store.ts

```diff
@@ -15,7 +15,11 @@ import { GraphQLError } from 'graphql';
 import jwtDecode from 'jwt-decode';
 
 const defaultOnRedirectCallback = (appState?: any): void => {
-  window.location.replace(appState?.returnTo || window.location.pathname);
```

Comment:
This was forcing a page load every time, whcih was necessary because the session wasn't available on the elements yet, which is why i check session in the file below